XavierGuertin/WildfireVisualizationProject
# #57-feat(frontend): Laying Groundwork for Front End Concept

### Summary
This PR introduces a new map component using OpenLayers to visualize wildfire data. The map includes various controls for zooming, panning, and toggling fullscreen mode.

### Changes Made
- Added `MapView` component using OpenLayers for map rendering.
- Integrated `Toolbar` component for map controls (zoom, pan, reset view, fullscreen).
- Updated `Navbar` to include a button for toggling the `Sidebar`.
- Implemented `Sidebar` component to manage map layers and datasets.
- `Sidebar` and `Toolbar` are draggable.
- Created a logo for wildfire project
- Added icons to the navbar

### Testing Instructions
1. Start the application using `npm start`.
2. Navigate to the main page.
3. Verify that the map is displayed correctly with the initial view centered at Ottawa.
4. Use the toolbar to zoom in, zoom out, reset the view, and pan the map in different directions.
5. Toggle fullscreen mode using the toolbar button.
6. Open the sidebar using the layers button in the navbar

### Screenshots (if applicable)
![image](https://github.com/user-attachments/assets/66ca7994-6c38-453b-9e55-2b5bb3b31a69)

### Additional Notes
The map was made using OpenLayers for rendering and managing map layers.
